sklearn/ensemble/forest.py

```diff
@@ -792,6 +794,9 @@ class RandomForestClassifier(ForestClassifier):
           `ceil(min_samples_split * n_samples)` are the minimum
           number of samples for each split.
 
+        .. versionchanged:: 0.18
+           a float value is percentage
```

Comment:
Nitpick: capitalize and fullstop. Maybe say "Added float values for percentages."?